src/IdeaStatiCa.BimApiLink/Plugin/FeaApplication.cs

```diff
@@ -14,6 +14,8 @@ namespace IdeaStatiCa.BimApiLink.Plugin
 {
 	public class FeaApplication : BimApiApplication
 	{
+		public IBimImporter BimImporter => _bimImporter;
+
 		private readonly IBimImporter _bimImporter;
```

Comment:
It is read only - no need to implement a property
mofify the name of the readonly property ang make it public